src/cdpTargetsProvider.ts

```diff
@@ -43,9 +43,7 @@ export default class CDPTargetsProvider implements vscode.TreeDataProvider<CDPTa
                         let targetsProcessed = 0;
                         responseArray.forEach(async (target: IRemoteTargetJson) => {
                             const actualTarget = fixRemoteWebSocket(hostname, port, target);
-                            if (actualTarget.type !== 'page') {
-                                targets.push(new CDPTarget(actualTarget, "", this.extensionPath));
-                            } else {
+                            if (actualTarget.type !== 'service_worker' && actualTarget.type !== 'shared_worker') {

```

Comment:
Is there a scenario where we want to use the extension for service worker debugging? Maybe with the network tab? If this is the case I think a setting like like:
"Show service workers on target list" and make it false by default makes more sense. Thoughts?

Reply:
I like that idea just so we don't break any existing workflows.  Will work on a fix now.

Reply:
Fixed.  See new GIF in description